lib/ExecutionEngine/LEVELZERORUNTIME/LevelZeroRuntimeWrappers.cpp

```diff
@@ -0,0 +1,304 @@
+// Copyright 2022 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <algorithm>
+#include <array>
+#include <atomic>
+#include <cassert>
+#include <cstdint>
+#include <cstdio>
+#include <cstdlib>
+#include <iostream>
+#include <tuple>
+#include <vector>
+
+#include <level_zero/ze_api.h>
+
+#ifdef _WIN32
+#define LEVEL_ZERO_RUNTIME_EXPORT __declspec(dllexport)
```

Comment:
cmake `generate_export_header` will do this for you

Reply:
building the library similar to upstream and hence we dont require to Include(GenerateExportHeader) ...its defined at the top of the file